### PR TITLE
Resolve  doc string bugs

### PR DESCRIPTION
**Pull request by ayushbisht2001**
#### Brief description of what is fixed or changed
Some of the doc string are falsely written. I have resolved some of the errors which occur during doc  testing. 

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by ayushbisht2001** [comment]
@asmeurer , please review my PR.

File: sympy/assumptions/refine.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def refine(expr, assumptions=True):
     x
 
     >>> refine(Q.real(x), Q.positive(x))
-    True
+    Q.real(x)
```
Comment (JSS95) [inline sympy/assumptions/refine.py:35]: `refine(Q.real(x), Q.positive(x))` returns `True` in current master branch.

Comment (ayushbisht2001) [reply sympy/assumptions/refine.py:35]: refine_error
 

@JSS95  But that's not correct .. If you run the code , it gives Q.real(x) not True..

File: sympy/discrete/transforms.py [hunk 2]
```diff
@@ -141,7 +141,7 @@ def _number_theoretic_transform(seq, prime, inverse=False):
                         "Number Theoretic Transform")
 
     a = [as_int(x) % p for x in seq]
-
+
```
Comment (0sidharth) [inline sympy/discrete/transforms.py:144]: Why have you added extra whitespace here? This causes flake8 code quality test to fail

File: sympy/assumptions/refine.py (hunk 1 above)
Comment (0sidharth) [reply sympy/assumptions/refine.py:35]: @ayushbisht2001 perhaps you are running `sympy-1.7.1` or an older version. I can confirm that on current master (1.8-dev), `True` is the result being returned. Further, if there was any discrepancy then I believe the doctests would be failing on current master.

**Comment by asmeurer** [comment]
Indeed, the `./bin/doctest` script automatically checks for incorrect examples. So if the doctests are passing, then this example should be working as it is written, unless there is some bug in the doctester.

**Comment by ayushbisht2001** [comment]
> Indeed, the `./bin/doctest` script automatically checks for incorrect examples. So if the doctests are passing, then this example should be working as it is written, unless there is some bug in the doctester.

@asmeurer I don't know why they are failing .  I am using latest **python 3.9.1** and **sympy  1.7.1**.   I run  ***python3 -m doctest -v file.py*** in command prompt and I found this  in many files : 

erros
:

**Comment by oscarbenjamin** [comment]
I don't see what issues this fixes. I think maybe there is a problem with your setup but the changes in this PR are not needed. I'm going to close this. For further help with these problems I suggest using the mailing list to have a discussion.

**Pull request closed without merging**